.github/actions/aws-ci/action.yml

```diff
@@ -14,7 +14,13 @@ inputs:
     description: 'AWS account type'
     required: false
     default: single-account
-
+  cloudbeat-config:
```

Comment:
If I understand correctly you are re-using the cspm action to asset inventory.

While I like it from a code re-usability stand point, I believe it could get confusing since vulnerability has its own action (correct?).

My point is, if previously we had a config file for each flavour, I'd rather add a new file to the new flavour, or merge them all in one. 

I do see that my suggestion adds more code and maintainability decreases. So I'm on the fence here. Anyway I'm placing the comment to give food for thought

Reply:
Let's go with a separate action for asset inventory. This will keep it more consistent and resolve the issue with parameterizing everything.